### PR TITLE
Remove old travis references

### PR DESCRIPTION
**Pull request by jarrodmillman**

### CODE REVIEW

File: doc/source/dev/testing.rst [hunk 1]
```diff
@@ -3,18 +3,17 @@
...
 
-The project is using `Travis-CI <https://travis-ci.org/PyWavelets/pywt>`_ service
-for continuous integration and testing.
+The project is using GitHub Actions for continuous integration and testing.
 
 Current build status is:
 
 .. image::
-    https://secure.travis-ci.org/PyWavelets/pywt.png?branch=master
+    https://github.com/PyWavelets/pywt/actions/workflows/tests/badge.svg?branch=master
```
Comment (rgommers) [inline doc/source/dev/testing.rst:14]: This link is broken, it should be https://github.com/PyWavelets/pywt/actions/workflows/tests.yml/badge.svg?branch=master I believe.

File: README.rst [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 +---------------+-----------------+
 
-.. |travis_ci| image:: https://api.travis-ci.org/PyWavelets/pywt.svg?branch=master
-   :align: middle
-   :target: https://travis-ci.org/PyWavelets/pywt
-   :alt: TravisCI Status
+
+.. |ghactions_ci| image:: https://github.com/PyWavelets/pywt/actions/workflows/tests/badge.svg?branch=master
+   :alt: Build Status
+   :target: https://github.com/PyWavelets/pywt/actions/workflows/tests.yml?query=branch%3Amaster
```
Comment (rgommers) [inline README.rst:14]: Did you mean for this to be a text link to the actions page, or should this also be https://github.com/PyWavelets/pywt/actions/workflows/tests.yml/badge.svg?branch=master

**Review by rgommers — COMMENTED** [review]
Thanks @jarrodmillman! Cleaning up TravisCI leftovers is nice. There seems to be a minor problem with links to badges, can you check?

File: README.rst (hunk 2 above)
Comment (jarrodmillman) [reply README.rst:14]: Fixed.

File: doc/source/dev/testing.rst (hunk 1 above)
Comment (jarrodmillman) [reply doc/source/dev/testing.rst]: Fixed.

**Review by rgommers — APPROVED** [review]
LGTM now, thanks Jarrod.

**Pull request merged**